docs/conf.py

```diff
@@ -161,16 +161,16 @@
     "dudir": ("http://docutils.sourceforge.net/docs/ref/rst/directives.html#%s", ""),
 }
 
+object_description_options_map = {}
+object_description_options_patterns = []
+
 # BEGIN: sphinx_immaterial.format_signatures extension options
```

Comment:
should probably move this comment (and the `# END: ...` comment) so the config option changes show up in the doc's `literalblock`.

Reply:
Oh, I see what you've done to the docs now. Please disregard this suggestion.

However, using `append()` in an example snippet implies you can append to the default value (without declaring it like you have done properly here).

Reply:
Yeah unfortunately that is not possible.  But we have done the same with appending to extensions.  Not sure what the best approach is.

Reply:
(The reason it is not possible is that the config file has to be evaluated in order to determine which extensions to load, but extensions can define config options, so we can't know what the defaults are until after loading the config)

Reply:
I'm resolving this as my main concern here is addressed by [this review comment](https://github.com/jbms/sphinx-immaterial/pull/75/files#r858192878).